nucypher/config/node.py

```diff
@@ -28,43 +31,63 @@ def __init__(self,
 
                  checksum_address: str = None,
                  is_me: bool = True,
-                 federated_only: bool = False,
-                 network_middleware=None,
+                 federated_only: bool = None,
```

Comment:
This is weird, why not make it `False`? `bool(None) == False`

Reply:
it's the expected type, and It can be set to None by a subclass method... this "override passthrough" can be tricky stuff.

Reply:
:/